src/main/webapp/app/exercises/shared/exercise-detail-common-actions/non-programming-exercise-detail-common-actions.component.html

```diff
@@ -12,19 +12,29 @@
         </a>
 
         <!-- Participations  -->
-        <a *ngIf="exercise.course" [routerLink]="baseResource + 'participations'" class="btn btn-primary btn-sm me-1" style="margin-bottom: 10px">
+        <a
+            *ngIf="exercise.course || exercise.exerciseGroup?.exam?.course"
```

Comment:
could be reduced to `course` if course would be an input variable.

Reply:
Yess 👍